.github/workflows/ci-test-backend.yml

```diff
@@ -50,22 +52,19 @@ jobs:
           curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $GITHUB_WORKSPACE v1.20.0
           go get -u github.com/mattn/goveralls
 
-      - name: run backend linters
+      - name: run backend linters for backend
```

Comment:
Looks unnecessary.

Reply:
I mean, it had “backend” word twice. Only one can live:)

Reply:
Remove `backend` in end